blackbox_repository/repository.py

```diff
@@ -0,0 +1,62 @@
+import logging
+from pathlib import Path
+from typing import List
+
+import s3fs as s3fs
+
+from blackbox_repository import BlackboxOffline
+from blackbox_repository.blackbox_offline import deserialize as deserialize_offline
+from blackbox_repository.blackbox_tabular import deserialize as deserialize_tabular
+
+# where the blackbox repository is stored on s3
+from blackbox_repository.conversion_scripts.recipes import generate_blackbox_recipe
+from blackbox_repository.conversion_scripts.utils import repository_path, s3_blackbox_folder
+
+
+def blackbox_list() -> List[str]:
+    """
+    :return: list of blackboxes available
+    """
+    return list(generate_blackbox_recipe.keys())
+
+
+def load(name: str, skip_if_present: bool = True) -> BlackboxOffline:
```

Comment:
Is the return type correct?
Also, for 'nas201', it seems a dict is returned? Or do you implement '[]' in order to select the dataset?

Reply:
No it was not thanks, I updated with the correct type.